docs-en/chapter_preface/suggestions.md

```diff
@@ -2,223 +2,225 @@
    
-    // Content comments for detailed code solutions.
+    // Content comments for details.
    
     /**
-     * multi-line
-     * marginal notes
+     * Multiline
+     * comments
      */
     ```
 
 === "Dart"
 
     ```dart title=""
-    /* Header comments for labeling functions, classes, test samples, etc. */
+    /* Header comments for labeling functions, classes, test samples, etc */
    
-    // Content comments for detailed code solutions.
+    // Content comments for details.
    
     /**
-     * multi-line
-     * marginal notes
+     * Multiline
+     * comments
      */
     ```
 
 === "Rust"
 
     ```rust title=""
-    /* Header comments for labeling functions, classes, test samples, etc. */
+    /* Header comments for labeling functions, classes, test samples, etc */
 
-    // Content comments for detailed code solutions.
+    // Content comments for details.
    
     /**
-     * multi-line
-     * marginal notes
+     * Multiline
+     * comments
      */
     ```
 
 === "C"
 
     ```c title=""
-    /* Header comments for labeling functions, classes, test samples, etc. */
+    /* Header comments for labeling functions, classes, test samples, etc */
    
-    // Content comments for detailed code solutions.
+    // Content comments for details.
    
     /**
-     * multi-line
-     * marginal notes
+     * Multiline
+     * comments
      */
     ```
 
 === "Zig"
 
     ```zig title=""
-    // Header comments for labeling functions, classes, test samples, etc.
+    // Header comments for labeling functions, classes, test samples, etc
    
-    // Content comments for detailed code solutions.
+    // Content comments for details.
    
-    // Multi-line
-    // Annotation
+    // Multiline
+    // comments
     ```
 
-## Learn Efficiently In Animated Graphic Solutions
+## Efficient Learning via Animated Illustrations
 
-Compared with text, videos and pictures have a higher degree of information density and structure and are easier to understand. In this book, **key and difficult knowledge will be presented mainly in the form of animations and graphs**, while the text serves as an explanation and supplement to the animations and graphs.
+Compared with text, videos and pictures have a higher density of information and are more structured, making them easier to understand. In this book, **key and difficult concepts are mainly presented through animations and illustrations**, with text serving as explanations and supplements.
 
-If, while reading the book, you find that a particular paragraph provides an animation or a graphic solution as shown below, **please use the figure as the primary source and the text as a supplement and synthesize the two to understand the content**.
+When encountering content with animations or illustrations as shown in the figure below, **prioritize understanding the figure, with text as supplementary**, integrating both for a comprehensive understanding.
```

Comment:
"When encountering content with animations or illustrations as shown in the figure below, **prioritize understanding the figure, with text as supplementary**, integrating both for a comprehensive understanding."
can be changed or added to ->
"When encountering content with animations or illustrations as shown in the figure below, **prioritize understanding the figure first, using the text as supplementary context**, integrating both for a comprehensive understanding."